editor/editor_dock_manager.cpp

```diff
@@ -52,6 +53,115 @@ enum class TabStyle {
 
 EditorDockManager *EditorDockManager::singleton = nullptr;
 
+// Implemented in input(..) as nearly every single child consumes the mouse input.
+void EditorDockTabContainer::input(const Ref<InputEvent> &p_event) {
+	ERR_FAIL_COND(p_event.is_null());
+
+	Ref<InputEventMouse> me = p_event;
+	if (me.is_valid() && is_dragging_dock) {
+		if (mouse_inside != get_global_rect().has_point(me->get_position())) {
+			mouse_inside = !mouse_inside;
+			queue_redraw();
+		}
+	}
+}
+
+void EditorDockTabContainer::_notification(int p_what) {
+	switch (p_what) {
+		case EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED: {
+			int _corner_radius = EDSCALE * EDITOR_GET("interface/theme/corner_radius").operator int();
+			if (_corner_radius != dock_drop_highlight->get_corner_radius(CORNER_TOP_LEFT)) {
```

Comment:
Remove leading underscore for variable name:

```suggestion
			int corner_radius = EDSCALE * EDITOR_GET("interface/theme/corner_radius").operator int();
			if (corner_radius != dock_drop_highlight->get_corner_radius(CORNER_TOP_LEFT)) {
```